software/base/src/test/java/org/apache/brooklyn/entity/brooklynnode/BrooklynNodeIntegrationTest.java

```diff
@@ -578,7 +578,7 @@ public void testUsesLocation() throws Exception {
         assertEquals(parseJson(locatedLocationsContent, ImmutableList.of(appLocationId, "longitude"), Double.class), 45.6, 0.00001);
     }
 
-    @Test(groups="Integration")
+    @Test(groups={"Integration", "Broken"})
```

Comment:
Didn't investigate. I expected that `BrooklynNode` is broken, but that's probably when using it with Karaf.
I'll have another look if it's working for you.